src/Microsoft.Azure.WebJobs.Host/Timers/DefaultWebJobsExceptionHandlerFactory.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.Hosting;
```

Comment:
This file (and other new files) missing file headers

Reply:
Fixed all missing headers.